apps/condo/domains/contact/components/CreateContactForm.tsx

```diff
@@ -75,6 +78,19 @@ export const CreateContactForm: React.FC = () => {
 
     const { loading, obj: property } = Property.useObject({ where:{ id: selectedPropertyId ? selectedPropertyId : null } })
 
+    const {
+        loading: isRolesLoading,
+        count: totalRoles,
+        objs: roles,
+    } = ContactRole.useObjects({
+        where: {
+            OR: [
+                { organization_is_null: true },
+                { organization: { id: get(organization, 'id') } },
```

Comment:
please check that if `organization` is `undefined`, then we will not receive redundant contacts

Reply:
This query receives contact roles. In the case organization is undefined we receive just common roles.